core/src/config_handler/Module.md

```diff
@@ -0,0 +1,5 @@
+Prints "Hello World!" with a main function.
```

Comment:
Please remove these

Reply:
Didn't see that oops. Fixed with https://github.com/yashodgayashan/environment_platform/pull/38/commits/bf1b471315f74ea4bc7d8e0173f613b7f743c821.